pkg/controller/installation/core_controller.go

```diff
@@ -657,6 +657,14 @@ func fillDefaults(instance *operator.Installation) error {
 		}
 	}
 
+	// Only fill default policy setup timeout if CalicoCNI is used.
```

Comment:
nit, but "CalicoCNI" is neither a valid enum option in our API nor a commonly used representation of "Calico CNI plugin", so saying "if the Calico CNI plugin is used" would be more clear

Reply:
Similarly, this comment is self-explanatory by reading the code (or rather, it would be if the code didn't actually do something different from what the comment says!) and doesn't really add any additional context. Would be better if the comment explained _why_ we only default in this case.

> (or rather, it would be if the code didn't actually do something different from what the comment says!) 

We also are checking the LinuxDataplane here, so the comment doesn't match the code :)

Reply:
Thanks - I have a personal goal to properly re-read all my comments after I amend the code beneath them :rofl:

Reply:
Considering removing the comment altogether, since as you say, it's redundant really, and the _why_ really only amounts to "because we haven't covered the other cases yet" :sweat_smile: